src/index.html

```diff
@@ -26,7 +27,7 @@ <h2>Add Your Score</h2>
                     <form action="#">
                         <input type="text" class="name" placeholder="Your Name">
                         <input type="text" class="score" placeholder="Your Score">
```

Comment:
- [Optional] Good job so far, Also it could be better to make the `type` attribute of the score to `number`, that way score will only be a number.

Reply:
@Tadesse-Alemayehu Thank you for this honest review. I will make the respective changes that you recommended😀